singularity-eos/eos/eos_noble_abel.hpp

```diff
@@ -91,8 +91,7 @@ class NobleAbel : public EosBase<NobleAbel> {
   template <typename Indexer_t = Real *>
   PORTABLE_INLINE_FUNCTION Real MinInternalEnergyFromDensity(
       const Real rho, Indexer_t &&lambda = static_cast<Real *>(nullptr)) const {
-    MinInternalEnergyIsNotEnabled("Noble Abel");
-    return 0.0;
+    return _qq;
```

Comment:
I figured I'd fix this in the Noble Able EOS while I was fixing this in the Carnahan Starling EOS